config/webpack/loaders/babel.js

```diff
@@ -4,11 +4,7 @@ const env = process.env.NODE_ENV || 'development';
 
 module.exports = {
   test: /\.js$/,
-  // include react-intl because transform-react-remove-prop-types needs to apply to it
-  exclude: {
-    test: /node_modules/,
-    exclude: /react-intl[\/\\](?!locale-data)/,
```

Comment:
There's no more need to add special behavior for react-intl. The reason we were doing this before was to explicitly remove the prop-types from react-intl while not breaking on other babel transforms being applied to it. Since the new loader only runs one babel transform, we can just apply it to all `node_modules`, whereas this loader doesn't apply at all to `node_modules`.